.gitignore

```diff
@@ -0,0 +1,9 @@
+*#
+/*.yaml
+*.exe
+tags
+
+prombench
```

Comment:
I would say lets just leave

```
prombench
cmd/fake-webserver/fake-webserver
cmd/scaler/scaler
```

any other ones that are local env specific can be added to `.git/info/exclude`